WearComplicationDataSourcesTestSuite/Wearable/src/main/java/com/example/android/wearable/wear/wearcomplicationproviderstestsuite/LongTextDataSourceService.kt

```diff
@@ -19,31 +19,32 @@ import android.app.PendingIntent
 import android.content.ComponentName
 import android.graphics.drawable.Icon
 import androidx.datastore.core.DataStore
-import androidx.wear.complications.ComplicationProviderService
-import androidx.wear.complications.ComplicationRequest
 import androidx.wear.complications.data.ComplicationData
 import androidx.wear.complications.data.ComplicationType
 import androidx.wear.complications.data.LongTextComplicationData
 import androidx.wear.complications.data.MonochromaticImage
 import androidx.wear.complications.data.PlainComplicationText
 import androidx.wear.complications.data.SmallImage
 import androidx.wear.complications.data.SmallImageType
+import androidx.wear.complications.datasource.ComplicationDataSourceService
+import androidx.wear.complications.datasource.ComplicationRequest
 
 /**
  * A complication provider that supports only [ComplicationType.LONG_TEXT] and cycles
  * through the possible configurations on tap.
  *
- * Note: This subclasses [SuspendingComplicationProviderService] instead of [ComplicationProviderService] to support
+ * Note: This subclasses [SuspendingComplicationDataSourceService] instead of [ComplicationDataSourceService] to support
  * coroutines, so data operations (specifically, calls to [DataStore]) can be supported directly in the
  * [onComplicationRequest].
- * See [SuspendingComplicationProviderService] for the implementation details.
+ * See [SuspendingComplicationDataSourceService] for the implementation details.
  *
  * If you don't perform any suspending operations to update your complications, you can subclass
- * [ComplicationProviderService] and override [onComplicationRequest] directly.
- * (see [NoDataProviderService] for an example)
+ * [ComplicationDataSourceService] and override [onComplicationRequest] directly.
+ * (see [NoDataDataSourceService] for an example)
  */
-class LongTextProviderService : SuspendingComplicationProviderService() {
+class LongTextDataSourceService : SuspendingComplicationDataSourceService() {
     override suspend fun onComplicationRequest(request: ComplicationRequest): ComplicationData? {
+        ComplicationDataSourceService
```

Comment:
Not sure what this line is for

Reply:
+1 does this compile?

Reply:
It does compile, funnily enough! (it's just a noop reference to a companion object)

Definitely a mistake, I think I was adding it to get the import for the KDoc.

Removed in 0d5dffd